app/code/Magento/Review/Block/Rating/Entity/Detailed.php

```diff
@@ -49,7 +49,7 @@ protected function _toHtml()
         $reviewsCount = $this->_ratingFactory->create()->getTotalReviews($entityId, true);
         if ($reviewsCount == 0) {
             #return __('Be the first to review this product');
-            $this->setTemplate('empty.phtml');
+            $this->setTemplate('Magento_Review::empty.phtml');
```

Comment:
Please, revert fix in places when the template specified dynamically

Reply:
Done.